x-pack/plugins/observability_solution/observability/public/navigation_tree.ts

```diff
@@ -222,137 +165,10 @@ export function createNavTree(pluginsStart: ObservabilityPublicPluginsStart) {
-              defaultMessage: 'Stack Management',
-            }),
-            renderAs: 'panelOpener',
-            spaceBefore: null,
-            children: [
-              {
-                title: 'Ingest',
-                children: [
-                  { link: 'management:ingest_pipelines' },
-                  { link: 'management:pipelines' },
-                ],
-              },
-              {
-                title: 'Data',
-                children: [
-                  { link: 'management:index_management' },
-                  { link: 'management:index_lifecycle_management' },
-                  { link: 'management:snapshot_restore' },
-                  { link: 'management:rollup_jobs' },
-                  { link: 'management:transform' },
-                  { link: 'management:cross_cluster_replication' },
-                  { link: 'management:remote_clusters' },
-                  { link: 'management:migrate_data' },
-                ],
-              },
-              {
-                title: 'Alerts and Insights',
-                children: [
-                  { link: 'management:triggersActions' },
-                  { link: 'management:cases' },
-                  { link: 'management:triggersActionsConnectors' },
-                  { link: 'management:reporting' },
-                  { link: 'management:jobsListLink' },
-                  { link: 'management:watcher' },
-                  { link: 'management:maintenanceWindows' },
-                ],
-              },
-              {
-                title: 'Security',
-                children: [
-                  { link: 'management:users' },
-                  { link: 'management:roles' },
-                  { link: 'management:api_keys' },
-                  { link: 'management:role_mappings' },
-                ],
-              },
-              {
-                title: 'Kibana',
-                children: [
-                  { link: 'management:dataViews' },
-                  { link: 'management:filesManagement' },
-                  { link: 'management:objects' },
-                  { link: 'management:tags' },
-                  { link: 'management:search_sessions' },
-                  { link: 'management:aiAssistantManagementSelection' },
-                  { link: 'management:spaces' },
-                  { link: 'management:settings' },
-                ],
-              },
-              {
-                title: 'Stack',
-                children: [
-                  { link: 'management:license_management' },
-                  { link: 'management:upgrade_assistant' },
-                ],
-              },
-            ],
-          },
-          {
-            link: 'integrations',
-          },
-          {
-            link: 'fleet',
-          },
           {
             id: 'machine_learning-landing',
             link: 'securitySolutionUI:machine_learning-landing',
```

Comment:
Apparently this link `securitySolutionUI` is responsible for the issue I describe [here](https://github.com/elastic/kibana/pull/192805#issuecomment-2359368851) regarding `Machine learning` menu not showing at all in o11y menu. I will remove the link completely. Let me know if there are any objections

Reply:
Fix in this [commit](https://github.com/elastic/kibana/pull/192805/commits/7c876466053d00f15a1f8a6260259c4eb49c5ecf).

Reply:
As discussed with @sebelga this is not the proper solution to this problem. I created a new [issue](https://github.com/elastic/kibana/issues/193526) with the solution proposal we discussed

Reply:
At o11y we agreed we don't want any landing pages, so we will keep the fix I provided in this [commit](https://github.com/elastic/kibana/pull/192805/commits/7c876466053d00f15a1f8a6260259c4eb49c5ecf)